MapleServer2/Types/Player.cs

```diff
@@ -87,19 +90,42 @@ public class Player {
             stats.MoveSpd = new PlayerStat(110, 100, 150);
             stats.JumpHeight = new PlayerStat(110, 100, 130);
 
-            return new Player
-            {
+            List<SkillTab> skillTabs = new List<SkillTab>();
+            skillTabs.Add(new SkillTab("test1"));
+
+            // Add runeblade skills set to level 1 and unlearned (normally would be loaded from database)
+            XmlDocument xmlDoc= new XmlDocument();
```

Comment:
Anything dealing with XML should definitely happen in another class

Reply:
Agree with this. Players have nothing to do with XML

Reply:
agree to this

Reply:
Disagree to this, unless there is an overarching XML parsing with more XML data, because this would only ever be parsed when creating a new character, so there is no reason to move it out of this class until it would be used more than 1 time.

Reply:
A class should only contain functionality that pertains to the objects it would create. As said above, XML should be its own separate class. We can call the method from here still if needed, but it should be broken out.